spec.html

```diff
@@ -20989,13 +20989,24 @@
             </tr>
             <tr>
               <td>
-                [[Evaluated]]
+                [[Status]]
               </td>
               <td>
-                Boolean
+                String
+              </td>
+              <td>
+                Initially `"uninstantiated"`. Can become `"instantiated"`, `"instantiating"`, `"evaluated"`, or `"errored"` as the module progresses throughout its lifecycle.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                [[ErrorCompletion]]
+              </td>
+              <td>
+                An abrupt completion
               </td>
               <td>
-                Initially *false*, *true* if evaluation of this module has started. Remains *true* when evaluation completes, even if it is an abrupt completion.
+                A completion record representing exception that occurred during instantiation or evaluation; meaningful only if [[Status]] is `"errored"`.
```

Comment:
Is there a benefit to having both "status" and "errorCompletion", as opposed to having something where "evaluated" would be a NormalCompletion, and "errored" an AbruptCompletion?

Given that Status has 5 states, and only three of them would be obvious in this model (`undefined`, ?, ?, NormalCompletion, AbruptCompletion), I'm not convinced this is the better path, but I wanted to raise the question - it seems odd to me to have a field that's only meaningful based on the value of another field.